arbitrum-docs/node-running/partials/_feed-relay-troubleshooting.md

```diff
@@ -0,0 +1,12 @@
+<table>
+  <tbody>
+      <tr>
+          <th style={{minWidth: 180 + 'px'}}>Scenario</th> 
+          <th>Solution</th>
+      </tr>
+      <tr>
+        <td>Waiting for peers / peer disconnected / no active peers: <code>TODO</code></td>
```

Comment:
This isn't rendered yet; it's a stub that we can pull into the troubleshooting page as we build out the node-type-specific content. 

I included it here to provide foundations; the troubleshooting page contains a commented-out demonstration of pulling it in.